spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/TaskCommands.java

```diff
@@ -65,7 +65,7 @@
 	private static final String PROPERTIES_OPTION = "properties";
 	private static final String PROPERTIES_FILE_OPTION = "propertiesFile";
 	private static final String ARGUMENTS_OPTION = "arguments";
-	private static final String PLATFORM_OPTION = "platformName";
+	private static final String PLATFORM_OPTION = "platform";
```

Comment:
In StreamCommands the option `platformName` is used, should probably switch to that to be consistent.  `platformName` is also used in `TasksLogsController`